src/caselawclient/models/identifiers/unpacker.py

```diff
@@ -0,0 +1,33 @@
+from lxml import etree
+
+from . import Identifier, InvalidIdentifierXMLRepresentationException
+from .neutral_citation import NeutralCitationNumber
+
+IDENTIFIER_NAMESPACE_MAP: dict[str, type[Identifier]] = {
```

Comment:
A little worried that this is out of the way for the Big List of Namespaces -- maybe this wants to live somewhere more central?

Reply:
The problem is that if we put it too central then we start to get circular import problems, which is why it's out here.